src/main/java/karuna/karuna_backend/email/EmailSendException.java

```diff
@@ -0,0 +1,8 @@
+package karuna.karuna_backend.email;
+
+public class EmailSendException extends RuntimeException {
+
+    public EmailSendException(String email) {
+        super("Can not send email to %s".formatted(email));
+    }
+}
```

Comment:
This should be in exception package as in other classes, move GlobalEmailExceptionHandler to that exception package as well, unless you think that is bad idea.